README.md

```diff
@@ -1,3 +1,5 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+- This project requires 2 .env files to be created: .env.test and .env.development.
+- In each of these files you should add: "PGDATABASE=YOUR_PGDATABASE_NAME_HERE", one line and without the quotation marks.
+- The .env.test should set the PGDATABASE to your test database and the .env.development should be set to your development database.
```

Comment:
As this is only dev and test data, and the repo is for people to try your code, you can give the actual database names here to make the setup process simpler.